_examples/real-world-examples/delayed-messages/main.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"context"
+	stdSQL "database/sql"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/google/uuid"
+	_ "github.com/lib/pq"
+
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill-sql/v3/pkg/sql"
+	"github.com/ThreeDotsLabs/watermill/components/cqrs"
+	"github.com/ThreeDotsLabs/watermill/components/delay"
+	"github.com/ThreeDotsLabs/watermill/message"
+)
+
+func main() {
```

Comment:
I wonder if we could not have a bit more real-life scenario here, like sending reminder when payment was not sent after X time or whatever?

or something what should happen after X after order was placed?